src/lib/graphqlUpload.ts

```diff
@@ -0,0 +1,28 @@
+import { graphqlUploadExpress } from "graphql-upload"
+import config from "../config"
+
+const {
+  PRODUCTION_ENV,
+  GRAPHQL_UPLOAD_MAX_FILES,
+  GRAPHQL_UPLOAD_MAX_FILE_SIZE_IN_BYTES,
+} = config
+
+export const graphqlUploadMiddleware = (req, res, next) => {
+  const accessToken = req.headers["x-access-token"] as string | undefined
+
+  if (PRODUCTION_ENV && req.is("multipart/form-data")) {
+    throw new Error("You cannot upload file for production")
+  }
```

Comment:
Acceptance Criteria has an item "Returns an error if queried against production". Maybe it's better not to throw an error (I'm afraid that `multipart/form-data` can be used somewhere else and such a check will break the logic there)?